db/reporting_structure.sql

```diff
@@ -22,50 +22,50 @@ SET row_security = off;
 
```

Comment:
@martha I think these to sql file changes should be backed out, and the warehouse_structure.sql file should be added, since that's the db that the migration affects

Reply:
Ok, thanks! To be honest, I often don't feel sure which .sql file diffs to include in my PRs..
Here I've backed out the changes from `db/structure.sql` and `db/reporting_structure.sql`, and kept the full changes that my migration generates against `db/warehouse_structure.sql`, although not all of those changes are related to this migration.